root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -123,6 +122,10 @@ const flags = require('../../flags');
     // because additional behavior isn't needed.
     MB.GuessCase.series = MB.GuessCase.work;
 
+    // Event doesn't have it's own handler, and just uses the work handler
+    // because additional behavior isn't needed.
```

Comment:
I would probably amend the Series one to say something like "Series and events don't …" and just add…